exporter/newrelicexporter/transformer.go

```diff
@@ -28,19 +28,22 @@ import (
 )
 
 const (
-	unitAttrKey               = "unit"
-	descriptionAttrKey        = "description"
-	collectorNameKey          = "collector.name"
-	collectorVersionKey       = "collector.version"
-	instrumentationNameKey    = "instrumentation.name"
-	instrumentationVersionKey = "instrumentation.version"
-	statusCodeKey             = "otel.status_code"
-	statusDescriptionKey      = "otel.status_description"
-	spanKindKey               = "span.kind"
-	spanIDKey                 = "span.id"
-	traceIDKey                = "trace.id"
-	logSeverityTextKey        = "log.level"
-	logSeverityNumKey         = "log.levelNum"
+	unitAttrKey                     = "unit"
+	descriptionAttrKey              = "description"
+	collectorNameKey                = "collector.name"
+	collectorVersionKey             = "collector.version"
+	instrumentationNameKey          = "instrumentation.name"
+	instrumentationVersionKey       = "instrumentation.version"
+	droppedAttributesCountKey       = "otel.dropped_attributes_count"
+	droppedEventsCountKey           = "otel.dropped_events_count"
```

Comment:
Is this a known convention? I cannot see them in the spec. If not part of the spec yet then submit a spec PR to add.

Reply:
@tigrannajaryan I'm not aware of a spec convention for these names. The closest thing that I found to it is in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/jaeger.md#events where it is describing how to convert an OTel span event to a Jaeger log and that the dropped_attributes_count on the span event should be mapped to an attribute with the name `otel.event.dropped_attributes_count`.

In this case we are converting an OTel span event to a New Relic span event. Is there a concern about there being a collision in the future with this name? Or is there a concern about New Relic using a name that's reserved for OTel? Or is this something that the spec is trying to standardize the name of for all exporters? Would the same concerns be present if the New Relic span event was created with the attribute `nr.dropped_attributes_count` (or something similar)? This information might help me to better understand what would need to be in a spec PR.

I also can't seem to find where in the spec repo to find conventions like this. The only relevant things that I found were some descriptions within the zipkin and jaeger exporter specs. Do you have any guidance there?

Reply:
I am not sure the spec for Jaeger is right to have a special field name for event dropped attributes but not other dropped attributes. I am going to raise an issue in the spec to clarify this.

`nr.dropped_attributes_count` is also not a recommended name since it can clash in the future. For vendor-specific attribute naming see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-and-label-naming.md#recommendations-for-application-developers 
The spec recommend prefixing with reverse domain name.

However, I think you don't need your own vendor-specific name for this. Otel must define a name for this.

Reply:
I submitted a spec issue to fix this https://github.com/open-telemetry/opentelemetry-specification/issues/1656

Reply:
Specification PR that clarifies this: https://github.com/open-telemetry/opentelemetry-specification/pull/1662

Reply:
Thank you @tigrannajaryan for getting the issue and PR submitted and moving forward.

Reply:
The spec PR is merged. Please fix this PR to follow the spec.

Reply:
@tigrannajaryan The attribute names should match the spec now.